index.html

```diff
@@ -30,16 +51,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" style="background: teal;">Box 1</div>
```

Comment:
We fixed it during our 1:1 standup but just leaving a comment here for reference. 
A better alternative to this is using a second class called `box1` instead of inline styling. As a website scales, it becomes more difficult to manage inline styles.